common/src/main/java/whocraft/tardis_refined/common/tardis/manager/TardisPilotingManager.java

```diff
@@ -626,7 +627,7 @@ public void crash() {
 
         this.setTargetPosition(landingLocation);
         TardisNavLocation landing = this.targetLocation;
-        var location =  findClosestValidPosition(landing);
+        var location = findClosestValidPosition(landing);
```

Comment:
No var.

Reply:
We need someone to go through and convert all of these. What is the purpose of typing the object vs var in this instance. Is it to ensure we get the expected output, even though the function has as specific return type?

Reply:
When updating between Minecraft versions, it becomes easier for one of us to port up the mod - we could easily spend hours looking for the correct object type, var is great outside of Minecraft but Minecraft changes so much in short amount of time